src/platform/site-wide/side-nav/components/NavItem.js

```diff
@@ -2,53 +2,42 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { get } from 'lodash';
+import classNames from 'classnames';
 // Relative
 import NavItemRow from './NavItemRow';
 import { NavItemPropTypes } from '../prop-types';
 
-/*
- Detects expanded children to modify the ending line style
- with a short delay.
- This is looks a bit tricky but ending line will have to render always first
- for level 1 items and then because of the recursive way of rendering children
- after this we can detect that children has been expanded and apply
- the style by finding it by its parentID.
-*/
-const shouldModifyEndingLine = (item, depth) => {
-  const expanded = get(item, 'expanded');
-  const hasChildren = get(item, 'hasChildren');
-  const shouldShowLineOpen = !!(depth === 2 && expanded && hasChildren);
-  if (shouldShowLineOpen) {
-    setTimeout(() => {
-      const element = document.getElementById(`${item.parentID}-line`);
-      if (element) {
-        element.className = 'line-open';
-      }
-    }, 1);
-  }
-  return null;
-};
-
-const NavItem = ({
-  depth,
-  item,
-  index,
-  renderChildItems,
-  sortedNavItems,
-  toggleItemExpanded,
-}) => {
+const NavItem = ({ depth, item, renderChildItems, toggleItemExpanded }) => {
   // Derive the item properties.
   const expanded = get(item, 'expanded');
   const hasChildren = get(item, 'hasChildren');
   const id = get(item, 'id');
+  const isSelected = get(item, 'isSelected');
```

Comment:
NIT: While we're changing this file, I wonder if we could use vanilla JS instead of `get` from `lodash`. I think it could produce fewer lines, and help move us closer to being able to delete our dependency on `lodash`. 

For example:
```
  const { expanded, hasChildren, id, isSelected } = item
  const isExpanded = expanded && depth === 2 && !isSelected;
  const moreThanLevel2SelectedExpanded = expanded && depth > 2 && isSelected;
```

Reply:
Agreed, @billfienberg  - update